src/richie/apps/core/admin.py

```diff
@@ -1,6 +1,13 @@
 """Utility functions for admin classes."""
```

Comment:
Will you document this part for third-party integration and remove this commit from your PR?

Reply:
I was planning to just remove it and add it to our cookie cutter. It is not necessary for 99% of the sites that will use Richie... I just added it here as a POC of what we need to do for our special use case in https://github.com/openfun/fun-mooc.